crates/invoker-api/src/invocation_reader.rs

```diff
@@ -73,10 +73,12 @@ pub trait InvocationReaderTransaction {
     type Error: std::error::Error + Send + Sync + 'static;
 
     /// Read the journal and the journal metadata for the given invocation id.
+    ///
+    /// When Option is none, no journal was found for the given invocation id.
```

Comment:
I think we return `None` if the invocation is not in state `InvocationStatus::Invoked`. This does not necessarily mean that there is no journal.

Reply:
Maybe the name of this method is not optimal and should rather be something like `read_journal_of_invoked_invocation` (quite a mouthful :-( ).

Reply:
Not gonna rename it with that atrocious name :D will add it to the docs